HerPublicWebsite/Views/HealthCheck/Index.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@using HerPublicWebsite.Controllers
+@{
+    ViewBag.Title = "Health check";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">Health check passed</h1>
+        <a href="@Url.Action(nameof(StaticPagesController.Index), "StaticPages")" role="button" draggable="false" class="govuk-button" data-module="govuk-button">
```

Comment:
FWIW we shouldn't really use StaticPagesController.Index as when we go live that page will be hosted on GDS rather than in our service.
I'll create a separate ticket to fix this up.

Reply:
Yep, makes sense. There are probably a few other places with the same issue, but hopefully by linking to the controller action itself they'll be easy to find!